components/english/english-speaking.tsx

```
@@ -0,0 +1,429 @@
+    definition: "Present, appearing, or found everywhere.",
+    translation: "Phổ biến, có mặt khắp nơi",
+    example: "Mobile phones are now ubiquitous in modern society.",
+    // synonyms: ["omnipresent", "universal", "pervasive"],
+    // antonyms: ["rare", "scarce", "limited"],
+  },
+];
+
+const currentCard = exampleCards[0]; // Replace with the actual current card logic
+
+export default function SpeakingPractice() {
+  const [filteredCards, setFilteredCards] = useState<VocabularyCard[]>([]);
```

ESLint

[error] 'setFilteredCards' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)